src/toasts/ServerLimitToast.tsx

```diff
@@ -25,12 +25,12 @@ const TOAST_KEY = "serverlimit";
 
 export const showToast = (limitType: string, onHideToast: () => void, adminContact?: string, syncError?: boolean) => {
     const errorText = messageForResourceLimitError(limitType, adminContact, {
-        'monthly_active_user': _td("Your homeserver has exceeded its user limit."),
+        'monthly_active_user': _td("This homeserver has hit its Monthly Active User limit."),
```

Comment:
Why use `this` instead of `your`?

Reply:
> Why use `this` instead of `your`?

Actually the lines below it like `This homeserver has been blocked by it's administrator.` and `This homeserver has exceeded one of its resource limits.` had word `this` :')

Reply:
> Why use `this` instead of `your`?
Should I change it to `your`?

Reply:
There might be some context behind this which I am not really aware of, so I would personally leave it as it was